### PR TITLE
Routing Forms - Avoid Flicker, Fix preload of Cal Font

### PR DESCRIPTION
**Pull request by hariombalhara**
## What does this PR do?

Fixes # (issue)
Issues that are fixed are explained in the [loom](https://www.loom.com/share/84ce5b5d1bfd4e39bb723750c683a7ed)

**Environment**: Staging(main branch)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by emrysal — APPROVED** [review]
Minor lint error, otherwise approved

**Review by pumfleet — APPROVED** [review]
Looks good to me, haven't tested

File: packages/app-store/ee/routing_forms/components/RoutingNavBar.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export default function RoutingNavBar({
   ];
   return (
     <div className="mb-4">
-      <NavTabs tabs={tabs} linkProps={{ shallow: true }} />
```
Comment (zomars) [inline packages/app-store/ee/routing_forms/components/RoutingNavBar.tsx:25]: Was this causing flicker?

File: apps/web/pages/_app.tsx [hunk 2]
```diff
@@ -27,6 +28,8 @@ import "../styles/globals.css";
 function MyApp(props: AppProps) {
   const { Component, pageProps, err, router } = props;
   let pageStatus = "200";
+  const { isReady, Theme } = useTheme("light");
```
Comment (zomars) [inline apps/web/pages/_app.tsx:31]: `isReady` is not being used

**Review by zomars — APPROVED** [review]

**Pull request merged**